operator/v1/types_insights.go

```diff
@@ -62,6 +62,9 @@ type GatherStatus struct {
 // insightsReport provides Insights health check report based on the most
 // recently sent Insights data.
 type InsightsReport struct {
+	// downloadedAt is the time when the last Insights report was downloaded
```

Comment:
Nit sentences should finish with a full stop

What does it mean for this to be empty?

Reply:
Updated. THanks

Reply:
Oh let me regenerate it...

Reply:
When is this field likely to be empty and what does it mean for it to be empty?

Reply:
The field will be empty until there's some Insights data upload and consecutive Insights analysis report download. For example it can be empty in clusters when the Insights operator is disabled (disconnected).

Reply:
Can you make sure to explain that within the godoc please